docs/listings/deps_rerun_t6.txt

```diff
@@ -1,22 +1,25 @@
 [ReFrame Setup]
-  version:           3.10.0-dev.3+149af549
+  version:           4.0.0-dev.1+36ffa085
   command:           './bin/reframe --restore-session --keep-stage-files -n T6 -r'
-  launched by:       user@host
+  launched by:       user@tresa.local
   working directory: '/home/user/Repositories/reframe'
-  settings file:     '<builtin>'
+  settings file:     '/home/user/Repositories/reframe/tutorials/config/settings.py'
```

Comment:
If I am not mistaken this should run without configuration file.

Reply:
Strange, I haven't change the `gendoclistings.py`. I will fix it.